libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -131,9 +133,10 @@ class AP_MotorsHeli_RSC {
     float           _collective_in;                 // collective in for throttle curve calculation, range 0-1.0f
     float           _rotor_rpm;                     // rotor rpm from speed sensor for governor
     float           _governor_disengage = 0.0f;     // throttle percentage where governor disenages to allow return to flight idle
-    float           _governor_droop_setting = 0.0f; // governor droop setting, range 0-100%
     float           _governor_output = 0.0f;        // governor output for rotor speed control
-    float           _governor_setpoint = 0.0f;      // governor speed setpoint, range 800-3500 rpm
+    float           _governor_range = 0.0f;         // RPM range +/- governor rpm reference setting where governor is operational
```

Comment:
There's actually no need to initialise these variables to zero I hear.  It just makes the code size a little bit larger than it needs to be.